src/lib/core/coio.c

```diff
@@ -96,23 +94,25 @@ coio_fill_addrinfo(struct addrinfo *ai_local, const char *host,
 {
 	ai_local->ai_next = NULL;
 	if (host_hint == 1) { // IPv4
-		ai_local->ai_addrlen = sizeof(sockaddr_in);
-		ai_local->ai_addr = (sockaddr*)malloc(ai_local->ai_addrlen);
+		ai_local->ai_addrlen = sizeof(struct sockaddr_in);
+		ai_local->ai_addr = xmalloc(ai_local->ai_addrlen);
```

Comment:
Nit: malloc -> xmalloc change is not listed in the commit message.

Reply:
Right. Mentioned it in the commit message.